src/NzbDrone.Core/Indexers/Definitions/Anilibria.cs

```diff
@@ -0,0 +1,341 @@
+            }
+
+            return title;
+        }
+
+        public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
+        {
+            var torrentInfos = new List<ReleaseInfo>();
+            var queryResponseItems = JsonConvert.DeserializeObject<List<AnilibriaTitle>>(indexerResponse.Content);
+
+            foreach (var tl in queryResponseItems)
+            {
+                foreach (var tr in tl.Torrents.List)
+                {
+                    var torrentInfo = new TorrentInfo
+                    {
+                        Title = composeTitle(tl, tr),
+                        InfoUrl = string.Format("{0}/release/{1}.html", _settings.BaseUrl.TrimEnd('/'), tl.Code),
+                        DownloadVolumeFactor = 0,
+                        UploadVolumeFactor = 1,
+                        Categories = new[]
+                        {
+                            NewznabStandardCategory.TVAnime
+                        },
+                        Seeders = tr.Seeders,
+                        Peers = tr.Leechers + tr.Seeders,
+                        Grabs = tr.Downloads,
+
+                        // API provides timestamp in UTC+3 timezone, so we need to substract 3 hours
+                        PublishDate = DateTimeUtil.UnixTimestampToDateTime(tr.UploadedTimestamp).AddHours(-3),
+                        Guid = _settings.BaseUrl + tr.Url,
+                        DownloadUrl = _settings.BaseUrl + tr.Url,
+                        Size = tr.TotalSize,
+                        Resolution = tr.Quality.Resolution,
+                        Codec = tr.Quality.Encoder
+                    };
+                    torrentInfos.Add(torrentInfo);
+                }
+            }
+
+            return torrentInfos.ToArray();
+        }
+
+        public Action<IDictionary<string, string>, DateTime?> CookiesUpdater { get; set; }
+    }
+
+    public class AnilibriaSettingsValidator : AbstractValidator<AnilibriaSettings>
+    {
+        public AnilibriaSettingsValidator()
+        {
+            RuleFor(c => c.ApiUrl).NotEmpty();
+        }
+    }
+
+    public class AnilibriaSettings : IIndexerSettings
+    {
+        private static readonly AnilibriaSettingsValidator Validator = new AnilibriaSettingsValidator();
+
+        public AnilibriaSettings()
+        {
+            ApiUrl = "https://api.anilibria.tv/";
+        }
+
+        [FieldDefinition(1, Label = "Base Url", Type = FieldType.Select, SelectOptionsProviderAction = "getUrls", HelpText = "Select which baseurl Prowlarr will use for requests to the site")]
+        public string BaseUrl { get; set; }
+
+        [FieldDefinition(2, Label = "Api Url", HelpText = "URL of api")]
+        public string ApiUrl { get; set; }
+
+        public NzbDroneValidationResult Validate()
+        {
+            return new NzbDroneValidationResult(Validator.Validate(this));
+        }
+    }
+
+    public class AnilibriaTitle
+    {
+        [JsonProperty("names")]
```

Comment:
no need to call out the JsonProperty if the property already matches the json (case is ignored)

Reply:
Removed JsonProperty where it's not needed. I used some generator site to generate this classes based on json example